swarm/network/hive.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/p2p"
 	"github.com/ethereum/go-ethereum/p2p/discover"
+	//"github.com/ethereum/go-ethereum/pot"
```

Comment:
Is this PR ready for merge or just to share? If this for merging, there is a lot of commented code which should be removed. Adding this comment only once

Reply:
It's VERY in progress :)

Reply:
so now time to remove commented code?